client/webpack.production.config.js

```diff
@@ -114,22 +115,17 @@ module.exports = {
     new ExtractTextPlugin('style-[name]-[chunkhash].css'),
     new HtmlWebpackPlugin({
       hash: true,
-      chunks: ['vendors', 'contrast-app'],
-      template: 'app/html/index.html',
-      filename: 'contrast.html'
-    }),
-    new HtmlWebpackPlugin({
-      hash: true,
       chunks: ['vendors', 'terminal-app'],
```

Comment:
I assume that right now the `contrast-theme` doesn't affect the `terminal-app` in any way, but do you think it would make sense to add it here now, so that we don't have to think about it in the future?